### ISSUE TITLE
Installation on Android TV (Amlogic X96mini, Android 7.1)

### ISSUE THREAD

**Issue by frumbert** [xbmc-catchuptv-au/repo#68]
Following the guide http://aussieaddons.com/installation/ let me install the `System -> Settings -> Add-ons -> Install from zip file` step. This completes without error. Any steps for apps beyond `System -> Settings -> Add-ons -> Aussie Add-ons ->` report "Failed to install a dependency".

Where to from here?

(p.s. iView etc have android native apps, but they require a touchscreen not a remote).

**Comment by andybotting** [comment]
Do you have any other repositories installed?

This is usually because another repository is taking over and failing to
provide some dependencies we require.

**Comment by frumbert** [comment]
Yes, but I didn't install them. It's  one of those smarttv boxes from ebay that come loaded with a tonne of crap preinstalled into kodi. I've been busily trying to remove it all. What's the easiest way to show you what is installed, or for me to determine which repo it's failing on?

**Comment by andybotting** [comment]
You may be able to find out from the log file.

You could try installing the Kodi log uploader to publish your log file
online after attempting the installation again. I think we should be able
to tell from that log which add-on and repository is giving us trouble.
